modules/network-observability-deploy-network-policy.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-deploy-network-policy_{context}"]
+= Configuring a network policy by using the FlowCollector custom resource
+You can configure the `FlowCollector` custom resource to deploy a Network Policy for Network Observability by setting the `spec.deployNetworkPolicy.enable` specification in  to `true`. By default, the specification is `false`.
+
+Ensure that Network Observability components you have configured can communicate with components that are external to the namespace, which might include:
+
+   * Connection to Loki (as defined in `FlowCollector` `spec.loki`)
+   * Connection to Kafka (as defined in `FlowCollector` `spec.kafka`)
+   * Connection to any exporter (as defined in FlowCollector `spec.exporters`)
+   * If you are using Loki and including it in the policy target, connection to an external object storage (as defined in your `LokiStack` related secret)
+
+If you need further isolation for egress traffic, you can create an additional policy that. For more information, see 
+//"Creating an egress network policy for Network Observability".
+
+.Procedure
+. . In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-network-policy_{context}"]
+.Configure `FlowCollector` for network policy
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  networkPolicy:
+    enable: true   <1>
+    additionalNamespaces: [] <2>
```

Comment:
I would leave the default values in the example, because if the user replaces them by an empty list, that would break a couple of things.

```suggestion
    additionalNamespaces: ["openshift-console", "openshift-monitoring"] <2>
```